website/blog/2021-04-02-tungsten.md

```diff
@@ -0,0 +1,369 @@
+---
+author: Tomasz Godzik
+title: Metals v0.10.1 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.1, which mainly concentrated
+on bugfixes together with a couple of smaller improvements. We also added
+support for the almost ready Scala 3 version 3.0.0-RC2. Its final release is
+coming soon and Metals will try to make it an amazing experience!
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">163</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">81</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">29</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">2</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/35?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Support for Scala 3.0.0-RC2
+- Smoother main and test discovery
+
+## Smoother main and test discovery
+
+Thanks to some great ideas (and obviously execution) by
+[@ckipp01](https://github.com/ckipp01) it is now possible to specify alternative
+debug parameters that enable users to:
+
+- Run main class in the current file, in case there are multiple main classes a
+  quick pick window will apear for the users to choose the exact main class to
+  run.
```

Comment:
```suggestion
- Run the main class in the current file, or in case that there are multiple main classes a
  quick pick window will appear for the users to choose the exact main class to
  run.
```

Reply:
Fixed!